### PR TITLE
Additional one services and stricter dependency 

### PR DESCRIPTION
**Pull request by sulaweyo**
- add econe and oneflow services
- add dependency opennebula to sunstone service
- add dependency mysqld to opennebula

The mysql dependency is not required if sqlite is used but i guess mysql would be more common than sqlite in the real world

### CODE REVIEW

File: opennebula.service [hunk 1]
```diff
@@ -1,8 +1,7 @@
 [Unit]
 Description=OpenNebula management
-After=network.target
-After=local-fs.target
-After=remote-fs.target
+After=network.target local-fs.target remote-fs.target mysqld.service 
+Requires=mysqld.service
```
Comment (rafamanzo) [inline opennebula.service:4]: I do agree MySQL is the common choice for production. But I'm not sure about requiring it for the service. Maybe just in the after is enough. Have a look how the ONE team has generated the systemd units for Debian 8: http://downloads.opennebula.org/repo/5.0/Debian/8/pool/opennebula/opennebula_5.0.2-2_amd64.deb

So, keeping the Arch philosophy of minimal installations, what do you think about removing this require?

**Comment by rafamanzo** [comment]
Indeed nice additions. I've asked just one question regarding the `mysqld.service` require.

Thank you very much!

ps.: I'm bumping to 5.0.2.

**Comment by sulaweyo** [comment]
For the minimal setup MySQL is not needed that's true. How about a second unit with mysql? That way the user can decide which to enable without editing the files. I'm not sure how to do the depend ond opennebula.service in that scenario...

**Comment by rafamanzo** [comment]
I believe having two versions for opennebula.service may be the way. What do you think?

**opennebula.service**:

```
[Unit]
Description=OpenNebula management
After=network.target local-fs.target remote-fs.target mysqld.service 

[Service]
User=oneadmin
Group=cloud
Type=forking
PermissionsStartOnly=true
ExecStartPre=/usr/bin/mkdir -p /var/run/one
ExecStartPre=/usr/bin/mkdir -p /var/lock/one
ExecStartPre=/usr/bin/chown -R oneadmin:cloud /var/run/one
ExecStartPre=/usr/bin/chown -R oneadmin:cloud /var/lock/one
ExecStart=/usr/bin/one start
ExecStop=/usr/bin/one stop

[Install]
WantedBy=multi-user.target
```

**opennebula-mysqld.service**:

```
[Unit]
Description=OpenNebula management
After=network.target local-fs.target remote-fs.target mysqld.service 
Requires=mysqld.service

[Service]
User=oneadmin
Group=cloud
Type=forking
PermissionsStartOnly=true
ExecStartPre=/usr/bin/mkdir -p /var/run/one
ExecStartPre=/usr/bin/mkdir -p /var/lock/one
ExecStartPre=/usr/bin/chown -R oneadmin:cloud /var/run/one
ExecStartPre=/usr/bin/chown -R oneadmin:cloud /var/lock/one
ExecStart=/usr/bin/one start
ExecStop=/usr/bin/one stop

[Install]
WantedBy=multi-user.target
```

**Comment by sulaweyo** [comment]
Looks good to me.
Maybe the 2 services should as well define a conflict with the other one (Conflicts=)

**Pull request merged**